resources/qml/TimelineRow.qml

```diff
@@ -71,27 +71,48 @@ Item {
         gesturePolicy: TapHandler.ReleaseWithinBounds
     }
 
-    RowLayout {
+    Control {
         id: row
+        property bool bubbleOnRight : isSender && Settings.bubbles
+        property int bubblePadding: (parent.width-(Settings.smallAvatars? 0 : Nheko.avatarSize+8))/10
+        anchors.rightMargin: isSender || !Settings.bubbles? 0 : bubblePadding
+        anchors.leftMargin: (Settings.smallAvatars? 0 : Nheko.avatarSize+8) + (bubbleOnRight? bubblePadding : 0) // align bubble with section header
+        anchors.left: bubbleOnRight? undefined : parent.left
+        anchors.right: bubbleOnRight? parent.right : undefined
+        property int maxWidth: parent.width-anchors.leftMargin-anchors.rightMargin
+        width: Settings.bubbles? Math.min(maxWidth,implicitWidth+metadata.width) : maxWidth
```

Comment:
```suggestion
        width: Settings.bubbles? Math.min(maxWidth, Math.max(implicitWidth, metadata.width)) : maxWidth
```

Or is this intentional?

![image](https://user-images.githubusercontent.com/16312324/153773540-c2db85cc-8371-4680-8e04-57ec7b65274b.png)